gcsio/src/test/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageFileSystemIntegrationTest.java

```diff
@@ -200,7 +200,9 @@ private void validateFileInfoInternal(
         .isEqualTo(expectedDirectory);
 
     if (expectedToExist) {
-      Instant currentTime = Instant.now();
+      // adding extra buffer time to overcome the flakiness observed
+      // because of clock skewness observed from GCS side. issue#834
```

Comment:
nit: I think that linking GitHub issue in the comment is not necessary as long as we linked it to PR/commit already

Reply:
Updated